components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -60,16 +60,29 @@ export type UserMenuGroup = {
 };
 
 export type UserMenuProps = HTMLAttributes<HTMLDivElement> & {
+    /** MUI Avatar that displays */
```

Comment:
Is this all we said on the RN one? I feel like we could be more informative here.